### PR TITLE
Collaboration

### PR DESCRIPTION
**Pull request by RyanRussell00**
I added a folder called 'getting_started' and it has 3 files:

- `Collaboration.md` =  Specifies how the participants can communicate and work with each other, as well as communicate with mentors
- `Software.md` = Accounts and Software that should be set up prior to the Hackathon
- `Preparation_List.md` = Things to do before the Hackathon and what to bring. It contains some useful resources to help prepare for your first hackathon.

By no means is this the final draft, please share ideas on what else we should add to any of these files, or if we should add any new files completely.

### CODE REVIEW

File: getting_started/Collaboration.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# **Table of Contents**
+
+## **Communication**
+
+## **Collaborative Coding**
+
+## **Collaborative Cloud**
```
Comment (etcadinfinitum) [inline getting_started/Collaboration.md]: Not necessary imo, but feel free to prove me wrong

File: getting_started/Collaboration.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# **Table of Contents**
```
Comment (etcadinfinitum) [inline getting_started/Collaboration.md]: Missing front matter for Jekyll generation; I'll demonstrate what I mean at the meeting today

File: getting_started/Preparation_List.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
+Whether you're a hackathon newbie or a 10x "Unicorn",
```
Comment (etcadinfinitum) [inline getting_started/Preparation_List.md]: Erica is working on the writeup for this file; see #5

File: getting_started/Software.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+#PLEASE ENSURE YOU HAVE INSTALLED AND SETUP THE FOLLOWING SOFTWARE AND ACCOUNTS
```
Comment (etcadinfinitum) [inline getting_started/Software.md]: This file _might_ be redundant, but we can discuss pros/cons as a group

**Review by etcadinfinitum — CHANGES_REQUESTED** [review]
Good start, but there's a few things that need to be changed

1. I appreciate the directory organization efforts, but the [Jekyll collections](https://jekyllrb.com/docs/collections/) schema that we are using requires all of the docs we want on the page to be dropped in the `_docs` folder, no subfolders. _I really appreciate the intent though :D_
2. Again, for Jekyll, we need the [front matter](https://jekyllrb.com/docs/front-matter/) at the top of the file. [The front matter we need for each file is detailed here.](https://raw.githubusercontent.com/UWB-ACM/Hackathon-Docs-2020/master/template.md)

**Comment by RyanRussell00** [comment]
-Add how to setup an AWS account and get free credits

**Comment by etcadinfinitum** [comment]
> -Add how to setup an AWS account and get free credits

See https://github.com/UWB-ACM/Hackathon-Docs-2020/blob/master/_docs/cloud_account_setup.md; link to that if you feel it necessary

File: getting_started/Collaboration.md (hunk 1 above)
Comment (RyanRussell00) [reply getting_started/Collaboration.md]: Removed

File: getting_started/Preparation_List.md (hunk 3 above)
Comment (RyanRussell00) [reply getting_started/Preparation_List.md]: Removed

File: getting_started/Software.md (hunk 4 above)
Comment (RyanRussell00) [reply getting_started/Software.md]: I agree, I just saw the file with all the info for this.

File: getting_started/Collaboration.md (hunk 2 above)
Comment (RyanRussell00) [reply getting_started/Collaboration.md]: Added the front matter, see latest push

File: _docs/collaboration.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+## **Collaborative Coding**
+To keep track of your code and prevent issues utilize the following:
+-[Git](git-scm.com) = Track changes and sync your code with others
+-[GitHub](Github.com) = Keep all your coding files in sync and help track changes in a Public or Private repo.
+*Note:* Github has something called "Projects" which is very much like Trello boards. So you can take advantage of these features!
+-[BitBucket.com](BitBucket.com) = Much like GitHub, but with private repositories.
+
+## **Collaborative Cloud**
+**TODO** Insert information about how to share AWS accounts and give permissions to others.
```
Comment (etcadinfinitum) [inline _docs/collaboration.md]:
```suggestion
Check out [our guide on choosing a cloud provider](https://docs.uwbhacks.com/cloud_setup.html) to pick a cloud platform to use for your project.

When you're ready, you can add users and give them permissions to use account services for your project. You can choose to share your administrator credentials, or you can create user accounts and give them specific permissions. When creating user accounts, you can follow the instructions for [AWS](https://docs.aws.amazon.com/marketplace/latest/userguide/marketplace-management-portal-user-access.html), [GCP](https://cloud.google.com/appengine/docs/standard/go/access-control#developers), and [Azure](https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/add-users-azure-active-directory).
```

File: _docs/collaboration.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
+---
+layout: default
+category: Getting Started
+title: Collaboration
+order: 1
```
Comment (etcadinfinitum) [inline _docs/collaboration.md:5]: This will appear after "Cloud Provider Setup", which makes sense to me.
```suggestion
order: 6
```

**Review by etcadinfinitum — APPROVED** [review]
See suggested changes. Otherwise, looks fine. Thanks!

**Pull request merged**